src/installer.ts

```diff
@@ -62,6 +62,7 @@ export async function getFlutter(
 
   core.exportVariable('FLUTTER_HOME', toolPath);
   core.addPath(path.join(toolPath, 'bin'));
+  core.addPath(path.join(toolPath, 'bin/cache/dart-sdk/bin'));
```

Comment:
'bin/cache/dart-sdk/bin' should be replaced by path.join to work with both *unix and windows

Reply:
Can you elaborate more on your suggestion? Did you mean something like:

```js
core.addPath(path.join(toolPath, 'bin', 'cache', 'dart-sdk', 'bin'));
```

Reply:
OK added a fix.

Reply:
cool